components/com_content/src/Controller/ArticleController.php

```diff
@@ -403,6 +404,67 @@ public function save($key = null, $urlVar = 'a_id')
 		return $result;
 	}
 
+	/**
+	 * Method to save article's title
+	 *
+	 * @return void
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	public function FEInlineEdition()
+	{
+		$this->checkToken();
+
+		$itemprop = $this->input->getString('itemprop');
+		$articleId = $this->input->getInt('a_id');
+
+		$newValue = '';
+
+		if ($itemprop == 'articleBody')
```

Comment:
Remove the else statement by placing the else statement before the if. It looks like this then:
```
$newValue = $this->input->getString('value');

if ($itemprop === 'articleBody')
{
	$newValue = $this->input->getRaw('value');
}
```

Always use strict comparison === instead of ==